python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml

```diff
@@ -1,24 +1,20 @@
 # openedx.yaml
```

Comment:
While we're here, can we add this comment to the top?
```
# This file describes this Open edX repo, as described in OEP-2:
# https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification
```

Reply:
Added the description as per suggestion.